marketplace.xml

```diff
@@ -11340,4 +11340,43 @@ Capabilities:
     </versions>
   </market_entry>
 
+  <market_entry>
+    <id>pentaho-cde-atolcd-plugins</id>
+    <type>Platform</type>
+    <name>Pentaho CDE Atol CD Plugins</name>
+    <category> 
+      <name>Dashboards</name>
+      <parent>
+        <name>Apps</name>
+      </parent>
+    </category>
+    <description>Provides an SVG Component for Community Dashboard Editor.</description>
+    <documentation_url>https://github.com/atolcd/pentaho-cde-atolcd-plugins/blob/master/README.md</documentation_url>
+    <author>Atol CD</author>
+    <author_url>http://www.atolcd.com</author_url>
+    <author_logo>http://docs.atolcd.com/docs/alfresco/guide_share/4.2/assets/images/logoAtolCD.png</author_logo>
+    <img>https://github.com/atolcd/pentaho-cde-atolcd-plugins/blob/master/docs/SVG%20Component/images/depPopulation.png</img>
+    <small_img>https://github.com/atolcd/pentaho-cde-atolcd-plugins/blob/master/docs/SVG%20Component/images/depPopulation.png</small_img>
+    <versions>
+      <version>
+        <branch>master</branch>
+        <version>1.0</version>
+        <name>1.0</name>
+        <package_url>https://github.com/atolcd/pentaho-cde-atolcd-plugins/releases/download/v1.0/pentaho-cde-atolcd-plugins-1.0.zip</package_url>
+        <description></description>
+        <changelog></changelog>
+        <build_id>1<build_id>
```

Comment:
This tag is not properly closed. Please review. You can run ant from the project root dir and it will validate the marketplace.xml for you.